elf/elf.cc

```diff
@@ -103,8 +103,14 @@ elf::elf(const std::shared_ptr<loader> &l)
         // Load sections
         const void *sec_data = l->load(m->hdr.shoff,
                                        m->hdr.shentsize * m->hdr.shnum);
+        if (NULL == sec_data) {
```

Comment:
Please follow the coding conventions of the surrounding code, which uses forward comparison: `sec_data == NULL`.